src/index.js

```diff
@@ -110,7 +110,7 @@ function login() {
 function loginUser(userName, password) {
   if (userName === 'manager') {
     loginAsManager()
-  } else if (userName.slice(0, 8) === 'customer' && parseInt(userName.slice(8)) <= 50) { //helper function for criteria? May need typeof === 
+  } else if (userName.slice(0, 8) === 'customer' && parseInt(userName.slice(8)) <= 50) { //helper function for criteria? May need typeof === 'number'
```

Comment:
Check this out!! https://stackoverflow.com/questions/3370263/separate-integers-and-text-in-a-string

I found it super useful for splitting up my username input into a string & integer- then I test each property separately.